### DISCUSSION TITLE
Posted on Level1Tech about Immich

### DISCUSSION THREAD

**Discussion by bikerchris** [immich-app/immich#14600] [Show and tell]
Hiya, I know I should put it here, but first put a comment about Immich over at Level1Tech.

Cheers and well done.

https://forum.level1techs.com/t/what-are-you-working-on-today/167475/3622?u=chrisa

**REVISION: Daniel kindly alerted me that the post was in a private section, so it's posted at the bottom of this Github thread.**

**Comment by danieldietzler** [comment]
That page seems to be private?

**Reply by bikerchris** [reply]
Very sorry, I thought it was public. I'll rePOST here.

**Comment by bikerchris** [comment]
COPY OF LEVEL1TECH POST, it was in response to another person using Immich:

Thought I’d mention I’ve been playing with #Immich too…mainly because I’ve been asked to go through over 60,000 photos across Apple/Android devices to find two specific people together. Sod that.

So far it’s been great on TrueNAS Scale…which itself has become so much better in recent releases. Incantations are no longer needed to install apps.

It’s interesting to see the database size

![Screenshot 2024-12-09 114610](https://github.com/user-attachments/assets/c6e1c4d1-c66c-4094-957b-2f53fd013863)

I used 3 different piles of photos and use the external storage option to connect them.

![Screenshot 2024-12-09 114645](https://github.com/user-attachments/assets/5fc40beb-59db-47b0-9866-476637b7a128)

Nearly finished now, I did up the active things, sometimes up to 100 and the CPU didn’t seem to mind.

![Screenshot 2024-12-09 114628](https://github.com/user-attachments/assets/0b4bbf66-56f3-4d39-8163-34d41e947cee)

Here’s the CPU activity/model (old machine), sometimes peaked at 99% but not often.

![Screenshot 2024-12-09 114557](https://github.com/user-attachments/assets/f08b6529-8da1-4ee5-922f-7d37b892b81a)

CPU Activity

![Screenshot 2024-12-09 115121](https://github.com/user-attachments/assets/b651dfd2-0c5b-451f-b029-599011284853)
![Screenshot 2024-12-09 115134](https://github.com/user-attachments/assets/a99cf2a1-ce3b-4412-89f8-b46ffa155f54)

Conclusion so far:

- It’s a bit fiddly to install, but you get there pretty quickly.
- May be exclude videos if you want to reduce database size.
- It’d say for a 4-6C+HT machine it takes around half a day to process 10k photos (no vid’s), including thumbnails, etc.
- I will very likely pay some money to them as they’ve done a great job.
- There’s a lot of moments when you say, “Oh I wish they had included this feature…”, and then you find they have.
- Under their advisement, I’m not depending on it and have installed it on a test machine with a test copy of the data.
- I gave it 64GB of RAM to help it along, with all 6 Cores and it didn’t get huffy. Also it didn’t exceed around 16GB of RAM usage.
